test/users.js

```diff
@@ -0,0 +1,107 @@
+const chai = require('chai'),
+  dictum = require('dictum.js'),
+  server = require('./../app'),
+  should = chai.should();
+
+describe('users', () => {
+  describe('/users POST', () => {
+    it('should be success', done => {
+      chai
+        .request(server)
+        .post('/users')
+        .send({
+          firstName: 'persona5',
+          lastName: 'apellido5',
+          password: 'falabella2017',
+          email: 'email5@wolox.com'
+        })
+        .then(res => {
+          res.should.have.status(200);
+          dictum.chai(res);
+        })
+        .then(() => done());
+    });
+    it('should fail because email exist', done => {
+      chai
+        .request(server)
+        .post('/users')
+        .send({
+          firstName: 'persona6',
+          lastName: 'apellido6',
+          password: 'falabella2017',
+          email: 'email5@wolox.com'
+        })
+        .catch(err => {
+          err.should.have.status(400);
+          err.response.should.be.json;
```

Comment:
idem here (with adding the test to check if the user was created), but with the negative case